src/databricks/labs/ucx/assessment/clusters.py

```diff
@@ -161,3 +165,42 @@ def snapshot(self) -> Iterable[ClusterInfo]:
     def _try_fetch(self) -> Iterable[ClusterInfo]:
         for row in self._fetch(f"SELECT * FROM {self._schema}.{self._table}"):
             yield ClusterInfo(*row)
+
+
+@dataclass
+class PolicyInfo:
+    policy_id: str
+    policy_name: str
+    policy_description: str | None = None
+    creator: str | None = None
+
+
+class PoliciesCrawler(CrawlerBase[PolicyInfo], CheckClusterMixin):
+    def __init__(self, ws: WorkspaceClient, sbe: SqlBackend, schema):
+        super().__init__(sbe, "hive_metastore", schema, "policies", PolicyInfo)
+        self._ws = ws
+
+    def _crawl(self) -> Iterable[PolicyInfo]:
+        all_policices = list(self._ws.cluster_policies.list())
+        return list(self._assess_policies(all_policices))
+
+    def _assess_policies(self, all_policices):
+        for policy in all_policices:
+            if policy.policy_id is None:
+                continue
+            policy_name = policy.name
+            creator_name = policy.creator_user_name
+            policy_info = PolicyInfo(
```

Comment:
and what does this give us? where do we analyse any UC incompatibilities here?

Reply:
Currently this table will store all the policy related info with spark version associated with policy. Users can compare the spark version of the policy with cluster dbr version and see if it can be upgraded or new policy has to be created

Reply:
It doesn't check/tell if this policy is compatible with UC or not, so it's useless. use `CheckClusterMixin` to inspect the logic of this policy for UC compatibility